test_conformance/buffers/test_buffer_fill.cpp

```diff
@@ -921,17 +921,15 @@ REGISTER_TEST(buffer_fill_int)
         log_info( "Testing random fill from offset %d for %d elements: \n", (int)offset_elements, (int)fill_elements );
 
         for ( i = 0; i < 5; i++ ){
+            volatile size_t j = 0;
```

Comment:
Why is `volatile` needed here?

Reply:
If we're changing the `j` declaration, i'd prefer to define `j` in each loops instead of using a variable that gets reused across multiple loops, i.e prefer `for(size_t j = ...`  That's more readable in my opinion.